src/DataAttribute.php

```diff
@@ -139,7 +139,7 @@ public function getValue($model): string
             $value = ArrayHelper::getValueByPath($model, $this->name);
         }
 
-        if (empty($value)) {
+        if (empty($value) && (int)$value !== 0) {
```

Comment:
```suggestion
        if ($value === null || $value === '') {
```

Seems, it's more clearly.

Reply:
`if (empty($value) && (int)$value !== 0)` === `if (false)`
If you're gonna rewrite as @vjik suggested, don't forget about other 'empty' values like `[]`, mb `false`

Reply:
Agree with empty array. 

For boolean values maybe print words "true" and "false" ?

Reply:
I could finish adopting the suggestions to merge it, thanks.